catalog/plugin/catalog-plugin-metacard-validation/src/main/java/ddf/catalog/metacard/validation/ValidityParamDelegate.java

```diff
@@ -0,0 +1,470 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+
+package ddf.catalog.metacard.validation;
+
+import java.util.Date;
+import java.util.List;
+
+import ddf.catalog.filter.FilterDelegate;
+
+public class ValidityParamDelegate extends FilterDelegate<Boolean> {
+
+    @Override
+    public Boolean and(List<Boolean> operands) {
+        return operands.stream().reduce(false, (a, b) -> a || b);
```

Comment:
It should be an OR. I added a comment explaining the and, or, and not filters in my newest commit.